src/lib/storage.ts

```diff
@@ -65,25 +104,25 @@ export default class AppStorage<Storage extends Record<string, any>/* Storage ex
     if(this.cache.hasOwnProperty(key)) {
       return this.getFromCache(key);
     } else if(this.useStorage) {
-      let value: any;
-      try {
-        value = await this.storage.get(key as string);
-        //console.log('[AS]: get result:', key, value);
-        //value = JSON.parse(value);
-      } catch(e) {
-        if(!['NO_ENTRY_FOUND', 'STORAGE_OFFLINE'].includes(e)) {
-          this.useStorage = false;
-          console.error('[AS]: get error:', e, key, value);
-        }
-      }
+      const r = this.getPromises.get(key);
+      if(r) return r;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
      if(r) {return r;}
```